src/neptune/internal/utils/runningmode.py

```diff
@@ -26,7 +26,7 @@ def in_interactive() -> bool:
 def in_notebook() -> bool:
     """Based on: https://stackoverflow.com/a/22424821/1565454"""
     try:
-        from IPython import get_ipython
+        from IPython import get_ipython  # type: ignore
```

Comment:
Are you sure this is needed?
I deleted this comment and ran `mypy --config=pyproject.toml src/neptune/internal/utils/runningmode.py`
and the result was `Success: no issues found in 1 source file`

Reply:
<img width="1011" alt="Screenshot 2024-06-03 at 09 11 41" src="https://github.com/neptune-ai/neptune-client/assets/917619/b02aea58-48a6-4ec4-a0df-165c0bbf7baf">